src/Tickets/Seating/Service/Layouts.php

```diff
@@ -234,6 +234,7 @@ public static function get_associated_posts_by_id( string $layout_id ): int {
 					LEFT JOIN %i AS layout_meta
 					ON posts.ID = layout_meta.post_id
 					WHERE posts.post_type IN ({$post_types})
+					AND posts.post_status IN ('publish', 'private')
```

Comment:
Are you sure these are the only post statuses to take into account?
Should this be excluding only trashed posts?